<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsListEdit</class>
 <widget class="QWidget" name="SettingsListEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QListView" name="listView">
     <property name="contextMenuPolicy">
      <enum>Qt::ActionsContextMenu</enum>
     </property>
     <property name="tabKeyNavigation">
      <bool>true</bool>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="buttonContainerWidget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QToolButton" name="moveUpButton"/>
      </item>
      <item>
       <widget class="QToolButton" name="moveDownButton"/>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>281</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QToolButton" name="addButton"/>
      </item>
      <item>
       <widget class="QToolButton" name="removeButton"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
  <action name="actionAdd">
   <property name="icon">
    <iconset resource="qsettingsdialog_standardsettingswidgets_res.qrc">
     <normaloff>:/QSettingsDialog/icons/standardSettingsWidgets/add.ico</normaloff>:/QSettingsDialog/icons/standardSettingsWidgets/add.ico</iconset>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
   <property name="toolTip">
    <string>Adds a new entry to the list</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Ins</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WidgetWithChildrenShortcut</enum>
   </property>
  </action>
  <action name="actionRemove">
   <property name="icon">
    <iconset resource="qsettingsdialog_standardsettingswidgets_res.qrc">
     <normaloff>:/QSettingsDialog/icons/standardSettingsWidgets/delete.ico</normaloff>:/QSettingsDialog/icons/standardSettingsWidgets/delete.ico</iconset>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
   <property name="toolTip">
    <string>Removes the selected entry from the list</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WidgetWithChildrenShortcut</enum>
   </property>
  </action>
  <action name="actionMove_Up">
   <property name="icon">
    <iconset resource="qsettingsdialog_standardsettingswidgets_res.qrc">
     <normaloff>:/QSettingsDialog/icons/standardSettingsWidgets/moveUp.ico</normaloff>:/QSettingsDialog/icons/standardSettingsWidgets/moveUp.ico</iconset>
   </property>
   <property name="text">
    <string>Move Up</string>
   </property>
   <property name="toolTip">
    <string>Moves the selected entry up in the list</string>
   </property>
   <property name="shortcut">
    <string>PgUp</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WidgetWithChildrenShortcut</enum>
   </property>
  </action>
  <action name="actionMove_Down">
   <property name="icon">
    <iconset resource="qsettingsdialog_standardsettingswidgets_res.qrc">
     <normaloff>:/QSettingsDialog/icons/standardSettingsWidgets/moveDown.ico</normaloff>:/QSettingsDialog/icons/standardSettingsWidgets/moveDown.ico</iconset>
   </property>
   <property name="text">
    <string>Move Down</string>
   </property>
   <property name="toolTip">
    <string>Moves the selected entry down in the list</string>
   </property>
   <property name="shortcut">
    <string>PgDown</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WidgetWithChildrenShortcut</enum>
   </property>
  </action>
 </widget>
 <resources>
  <include location="qsettingsdialog_standardsettingswidgets_res.qrc"/>
 </resources>
 <connections/>
</ui>
